.github/workflows/backend-ci.yml

```diff
@@ -5,10 +5,12 @@ on:
   push:
     paths:
       - 'backend/**'
+      - 'solidago/**'
       - '.github/workflows/backend-ci.yml'
   pull_request:
     paths:
       - 'backend/**'
+      - 'solidago/**'
       - '.github/workflows/backend-ci.yml'
```

Comment:
I updated this configuration in 5a5b96214e to run backend tests: as Tournesol backend is quite tightly coupled with Solidago, I thing it's useful to run those tests even on changes that are internal to Solidago.